signing_clients/apps.py

```diff
@@ -310,7 +310,7 @@ def make_signed(self, signature, outpath=None):
                 for f in sorted(zin.infolist()):
                     # Make sure we exclude any of our signature and manifest
                     # files
-                    if filename in RESIGN_IGNORE:
+                    if f.filename in RESIGN_IGNORE:
```

Comment:
Do you have any tests that fail now when you edit this to just ``filename`` like before this patch?